Resources/views/Collector/db.html.twig

```diff
@@ -103,38 +103,42 @@
     <h2>Query Metrics</h2>
 
     <div class="metrics">
-        <div class="metric">
-            <span class="value">{{ collector.querycount }}</span>
-            <span class="label">Database Queries</span>
-        </div>
-
-        <div class="metric">
-            <span class="value">{{ collector.groupedQueryCount }}</span>
-            <span class="label">Different statements</span>
-        </div>
-
-        <div class="metric">
-            <span class="value">{{ '%0.2f'|format(collector.time * 1000) }} ms</span>
-            <span class="label">Query time</span>
-        </div>
-
-        <div class="metric">
-            <span class="value">{{ collector.invalidEntityCount }}</span>
-            <span class="label">Invalid entities</span>
-        </div>
+        <div class="metric-group">
```

Comment:
What are these for? Is it so that it stretches to full width?

Reply:
It “groups” the metrics. Look at the screenshots: in 6.1 it is 4 blocks, in 6.2 it is 4 parts of a bigger block. Without it, there would be whitespace between the blocks